examples/http-tinygo-outbound-http/outbound-http-to-same-app/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	spinhttp "github.com/fermyon/spin/sdk/go/http"
+)
+
+func init() {
+	spinhttp.Handle(func(w http.ResponseWriter, r *http.Request) {
+		resp, err := spinhttp.Get("/hello")
```

Comment:
If convenient, could you try this as just `http.Get` ([like here](https://github.com/fermyon/spin/blob/main/sdk/go/integration_test.go#L26))?

Reply:
that worked as well